renovate.json

```diff
@@ -56,6 +56,18 @@
       "matchStrings": [
         "// renovate: datasource=(?<datasource>[a-z-.]+?) depName=(?<depName>[^\\s]+?)(?: (?:packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[^\\s]+?))?(?: extractVersion=(?<extractVersion>[^\\s]+?))?\\n[^\\n]+[A-Za-z0-9_]+?_VERSION\\s*=\\s*\"(?<currentValue>.+?)\";"
       ]
+    },
```

Comment:
Do we still need this, if we use the compose file for keycloak deployment?

Reply:
don't we need it for the compose file?